packages/db/src/models/base.ts

```diff
@@ -17,11 +17,19 @@ class BasePgModel<ItemType, RecordType extends { cumulus_id: number }> {
     this.tableName = tableName;
   }
 
-  get(
+  async get(
     knexOrTransaction: Knex | Knex.Transaction,
     params: Partial<RecordType>
   ) {
-    return knexOrTransaction<RecordType>(this.tableName).where(params).first();
+    const record = await knexOrTransaction<RecordType>(this.tableName)
+      .where(params)
+      .first();
+
+    if (!isRecordDefined(record)) {
+      throw new RecordDoesNotExist(`Record in ${this.tableName} with identifiers ${JSON.stringify(params)} does not exist.`);
+    }
+
+    return record;
   }
```

Comment:
This was briefly discussed in chat. We want to throw an error if we can't find the record we're trying to GET.